esn-frontend-calendar#24: Registered i18n translation tables

huy-ta opened the pull request:
Resolves #24.

The corrected translation files are taken from this PR: https://ci.linagora.com/linagora/lgs/openpaas/linagora.esn.calendar/merge_requests/664

- A quick demo to show that it works:

[![Demo-Calendar-i18n](https://user-images.githubusercontent.com/24670327/89752406-6dfbfa80-dafe-11ea-9f90-ec72faa08c72.gif)](https://streamable.com/nf1e09)

tuanlc commented on src/linagora.esn.calendar/app/app.config.js:
```diff
@@ -0,0 +1,20 @@
+require('./core/application-menu-calendar.directive.js');
```
missing `use strict`

huy-ta replied:
Why do we need it?

tuanlc commented on src/linagora.esn.calendar/app/app.config.js:
```diff
@@ -0,0 +1,20 @@
+  dynamicDirectiveServiceProvider.addInjection('esn-application-menu', dd);
+}
+
+// TODO: Write tests for this (https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/24)
```
We never write test for config blocks before ;) AFAIK

huy-ta replied:
Yes, you're right.

huy-ta commented:
> Calendar SPA contains `linagora.esn.resource` also. Did you check it?

I've just added a commit for `linagora.esn.resource`. The Chinese and Russian translations were missing though, so I added them using Google Translate.

tuanlc commented:
> The Chinese and Russian translations were missing though, so I added them using Google Translate.

You can ask for Russian translation on the translation channel. But yes, maybe no response this week. ;(

huy-ta commented:
> Why there is no content for `src/linagora.esn.resource/i18n/en.json` file?

Oops, forgot to press Save.

huy-ta commented:
> > The Chinese and Russian translations were missing though, so I added them using Google Translate.
> 
> You can ask for Russian translation on the translation channel. But yes, maybe no response this week. ;(

These are very simple words and sentences, so I think Google Translate can do the job pretty well.

piavgh commented on src/linagora.esn.calendar/app/app.config.js:
```diff
@@ -0,0 +1,21 @@
+  .config(registerI18N);
+
+function injectApplicationMenuDirective(dynamicDirectiveServiceProvider) {
+  var dd = new dynamicDirectiveServiceProvider.DynamicDirective(true, 'application-menu-calendar', {priority: 40});
```
This is new code so I guess we can use `const` here

piavgh replied:
And I'm OK with other files

huy-ta replied:
Well this part of the code will be removed due to https://github.com/OpenPaaS-Suite/esn-frontend-calendar/pull/43.

piavgh replied:
Ok :v
